com.github.rajsolai.textsnatcher.yml

```diff
@@ -0,0 +1,27 @@
+app-id: com.github.rajsolai.textsnatcher
+
+runtime: org.gnome.Platform
+runtime-version: "40"
+base: io.elementary.BaseApp
+base-version: "juno-20.08"
+sdk: org.gnome.Sdk
+command: com.github.rajsolai.textsnatcher
+
+finish-args:
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=/home"
+  - "--talk-name=org.gnome.Shell.Screenshot"
+  - "--system-talk-name=org.freedesktop.Accounts"
```

Comment:
why is this needed?

Reply:
[--system-talk-name=org.freedesktop.Accounts](https://valadoc.org/granite/Granite.Settings.prefers_color_scheme.html) Granite Needs this permission for prefers_colors_scheme method.

"--talk-name=org.gnome.Shell.Screenshot" is used to access screenshot dbus-service